### PR TITLE
Added initial setup for iOS

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: docs/getting-started/apple/setup.md [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+
+## Setting up fastlane
+
+Navigate your terminal to your project's directory and run
+
+```no-highlight
+fastlane init
+```
+
+You'll be asked for your Apple ID, and `fastlane` will automatically generate a fastlane configuration for you, based on the information provided.
```
Comment (mfurtak) [inline docs/getting-started/apple/setup.md]: `fastlane` or _fastlane_ ?

File: docs/getting-started/apple/setup.md [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+
+You can see the newly created `fastlane` directory, with all your existing metadata fetched from iTunes Connect.
+
+<img src="/img/getting-started/apple/fastlane-init.png" width="800" />
+
+The most interesting file is `fastlane/Fastfile`, which contains all the information that is needed to distribute your app.
+
+## What's next?
+
+`fastlane` created all the required files for you, now you can go ahead and customise it to generate screenshots or to automatically distribute new builds
```
Comment (mfurtak) [inline docs/getting-started/apple/setup.md]: `fastlane` or _fastlane_ ?

File: docs/getting-started/apple/setup.md [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+
+- [Automatically generate screenshots](screenshots.md)
+- [Distribute beta builds](beta-deployment.md)
+- [Deploy to the App Store](appstore-deployment.md)
+
+### Use a Gemfile
+
+It is recommended that you use a Gemfile to define your dependency to `fastlane`. This will clearly define the used fastlane version, and its dependencies, and will also speed up using fastlane.
```
Comment (mfurtak) [inline docs/getting-started/apple/setup.md]: `fastlane` or _fastlane_ ?

File: docs/getting-started/apple/setup.md [hunk 4]
```diff
@@ -1,7 +1,46 @@
...
+- Create a `Gemfile` with the content
+```ruby
+source "https://rubygems.org"
+
+gem "fastlane"
+```
+- Run `bundle update` and add both the `Gemfile` and the `Gemfile.lock` to version control
+- Every time you run `fastlane`, use `bundle exec fastlane [lane]`
+- On your CI, add `bundle install` as your first build step
+- To update `fastlane`, just run `bundle update`
```
Comment (mfurtak) [inline docs/getting-started/apple/setup.md]: `fastlane` or _fastlane_ ?

**Comment by mfurtak** [comment]
Looks good! Style questions:

- `fastlane` or _fastlane_ when referring to the tool but not the command?
- Do we want headers to be "Sentence case" or "Title Case"?

File: docs/getting-started/apple/setup.md (hunk 1 above)
Comment (KrauseFx) [inline docs/getting-started/apple/setup.md]: We haven't made a decision yet, right? https://github.com/fastlane/docs/issues/36

**Comment by KrauseFx** [comment]
@mfurtak Re headers: I'm ok either way, however the tools names are lower case. So Title Case might look weird with just the fastlane tools being lower case

**Pull request merged**